sdkbuild/python.go

```diff
@@ -0,0 +1,154 @@
+package sdkbuild
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+	"strconv"
+	"strings"
+)
+
+// BuildPythonProgramOptions are options for BuildPythonProgram.
+type BuildPythonProgramOptions struct {
+	// Directory that will have a temporary directory created underneath. This
+	// should be a Poetry project with a pyproject.toml.
+	BaseDir string
+	// Required version. If it contains a slash it is assumed to be a path with
+	// a single wheel in the dist directory. Otherwise it is a specific version
+	// (with leading "v" is trimmed if present).
+	Version string
+	// Required Poetry dependency name of BaseDir.
+	HarnessDependency string
```

Comment:
Same as my TS comment (below), name this `AdditionalDependencies` or something similar.

Reply:
I am trying to not be more flexible than necessary. I can change the term to `ImplementationDependency` or something, but there are no backwards compatibility requirements here forcing me to future proof this for multiple dependency names future that doesn't exist (until it does).